sources/osg/WebGLCaps.js

```diff
@@ -273,13 +260,13 @@ WebGLCaps.prototype = {
         return status;
     },
     hasLinearHalfFloatRTT: function ( gl ) {
-        return this._webGLExtensions[ 'OES_texture_half_float_linear' ] && this.checkSupportRTT( gl, Texture.HALF_FLOAT, Texture.LINEAR );
+        return this._webGLExtensions[ 'OES_texture_half_float_linear' ] && this.checkSupportRTT( gl, Texture.HALF_FLOAT_OES, Texture.LINEAR );
     },
     hasLinearFloatRTT: function ( gl ) {
         return this._webGLExtensions[ 'OES_texture_float_linear' ] && this.checkSupportRTT( gl, Texture.FLOAT, Texture.LINEAR );
     },
     hasHalfFloatRTT: function ( gl ) {
-        return this._webGLExtensions[ 'OES_texture_half_float' ] && this.checkSupportRTT( gl, Texture.HALF_FLOAT, Texture.NEAREST );
+        return this._webGLExtensions[ 'OES_texture_half_float' ] && this.checkSupportRTT( gl, Texture.HALF_FLOAT_OES, Texture.NEAREST );
```

Comment:
The function name already implied that is it gl1/gl2 agnostic, it just tells you if can render to a h/float texture.